v63003/outref/gtm8846.txt

```diff
@@ -0,0 +1,259 @@
+%YDB-I-DBFILECREATED, Database file ##TEST_PATH##/mumps.dat created
+# -----------------------------------------
+# 75 slashes
+# -----------------------------------------
+# Verifying no Journal Files before running Mupip Set
+##TEST_COM_PATH##/lsminusl.csh: No match.
+##TEST_COM_PATH##/lsminusl.csh: No match.
+# Paths we are feeding to Mupip Set Command
+.///////////////////////////////////////////////////////////////////////////abcd.mjl
+temp///////////////////////////////////////////////////////////////////////////abcd.mjl
+%YDB-I-JNLCREATE, Journal file ##TEST_PATH##/abcd.mjl created for region DEFAULT with BEFORE_IMAGES
+%YDB-I-JNLSTATE, Journaling state for region DEFAULT is now ON
+%YDB-I-JNLCREATE, Journal file ##TEST_PATH##/temp/abcd.mjl created for region DEFAULT with BEFORE_IMAGES
+%YDB-I-JNLSTATE, Journaling state for region DEFAULT is now ON
+# Verifying journal file was made properly
+-rw-rw-rw- 1 vinay gtc abcd.mjl
```

Comment:
ls -l output has userid, group and size information which can vary across multiple userids and YottaDB releases. Just use "ls" instead of "ls -1" which will just print the filename minus the other columns.